eb_hooks.py

```diff
@@ -54,13 +54,25 @@ def get_rpath_override_dirs(software_name):
 
     return rpath_injection_dirs
 
+def set_Pillow_envvars(ec):
+    """Get an EESSI_CPATH environment variable from the environment"""
+    EESSI_CPATH = os.getenv('EESSI_EPREFIX') + '/usr/include'
+    EESSI_LIB_PATH = os.getenv('EESSI_EPREFIX') + '/usr/lib64'
+    if ec.name == 'Pillow':
+        os.environ['CPATH'] = os.pathsep + EESSI_CPATH
+        os.environ['LIBRARY_PATH'] = os.pathsep + EESSI_LIB_PATH
+        print_msg("NOTE: For Pillow which has zlib as a dependancy, CPATH has been set to %s", os.getenv('CPATH'))
+        print_msg("NOTE: For Pillow which has zlib as a dependancy, LIBRARY_PATH has been set to %s", os.getenv('LIBRARY_PATH'))
+        ec.log.info("NOTE: For Pillow which has zlib as a dependancy, CPATH has been set to %s", os.getenv('CPATH'))
+        ec.log.info("NOTE: For Pillow which has zlib as a dependancy, LIBRARY_PATH has been set to %s", os.getenv('LIBRARY_PATH'))
+
 
 def parse_hook(ec, *args, **kwargs):
```

Comment:
I'm not sure here is the right place to do this, have you checked that the envvars are still set during the configure step? I suspect that the configure step is the right place to do this.